simplemma/rules.py

```diff
@@ -499,3 +488,18 @@ def apply_ru(token: str) -> Optional[str]:
         if token.endswith(ending):
             return token[: -len(ending)] + base
     return None
+
+
+PREFIXES: Dict[str, Callable[[str], Optional[str]]] = {
+    "de": apply_prefixes_de,
+    "ru": apply_prefixes_ru,
+}
+
+RULES: Dict[str, Callable[[str, bool], Optional[str]]] = {
```

Comment:
I would prefer a variable name suggesting that it's something dynamic, it isn't just a dictionary. We could keep `APPLY_RULES`, don't you think?